indico/cli/admin.py

```diff
@@ -70,7 +70,7 @@ def user_search(substring, include_deleted, include_pending, include_external, *
     assert set(criteria.viewkeys()) == {'first_name', 'last_name', 'email', 'affiliation'}
     criteria = {k: v for k, v in criteria.viewitems() if v is not None}
     res = search_users(exact=(not substring), include_deleted=include_deleted, include_pending=include_pending,
-                       external=include_external, **criteria)
+                       external=include_external, allow_system_user=True, **criteria)
```

Comment:
Which [is already the case](Already the case: https://github.com/indico/indico/pull/2933/files#diff-6a85a90c99fa83a08eb0eed05308f410R174) - this line here is just from the user search on the CLI